pkg/pillar/cmd/volumemgr/appandimage.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2020 Zededa, Inc.
```

Comment:
Is this a move from zedmanager, an identical duplicate file, or something else?

Aren't the keys different for latching the CT?
Should the function names etc be different?

Reply:
Changed function names. Thanks for pointing it out.